### PR TITLE
Feature/reports tests

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Adds tests to reports module

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: tests/_support/Step/Acceptance/Reports.php [hunk 1]
```diff
@@ -51,6 +51,8 @@ public function createAccount($name)
         $Sidebar->clickSideBarAction('Create');
         $I->waitForEditViewVisible();
         $I->fillField('#name', $name);
+        $I->fillField('#phone_office', '(810) 267-0146');
+        $I->fillField('#website', 'www.afakeurl.com');
```
Comment (horus68) [inline tests/_support/Step/Acceptance/Reports.php]: better to use www.example.com/ in examples!

Comment (Dillon-Brown) [reply tests/_support/Step/Acceptance/Reports.php]: :+1: will replace with fzaninotto/faker for generating fake data.

File: tests/acceptance/modules/AOR_Reports/AOR_ReportsCest.php [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+        }
+
+        $this->fakeDataSeed = rand(0, 2048);
+        $this->fakeData->seed($this->fakeDataSeed);
+    }
+
+    /**
+     * @param AcceptanceTester $I
+     */
+    public function _after(AcceptanceTester $I)
```
Comment (gymad) [inline tests/acceptance/modules/AOR_Reports/AOR_ReportsCest.php]: Are there any reason because empty method needs here?

Comment (Dillon-Brown) [reply tests/acceptance/modules/AOR_Reports/AOR_ReportsCest.php]: Removed

**Pull request merged**